Journaley/Utilities/PostMarkdownParser.cs

```diff
@@ -0,0 +1,114 @@
+﻿namespace Journaley.Utilities
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Text.RegularExpressions;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// Helper class for fixing MarkdownDeep parsed HTML.
+    /// </summary>
+    public class PostMarkdownParser
+    {
+        /// <summary>
+        /// Perform fixes after MarkdownDeep parsing for publishing
+        /// to Journaley.
```

Comment:
You need to be more explicit here and say what it fixes.
1. automatically add `<br />` tags where the user hit enter.
2. add strikethrough style support